src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionTestResultViewModel.swift

```diff
@@ -128,6 +135,9 @@ class ExposureSubmissionTestResultViewModel {
 	private func updateForCurrentTestResult() {
 		self.dynamicTableViewModel = DynamicTableViewModel([currentTestResultSection])
 		updateButtons()
+		
+		// Also update the warn others model
+		self.warnOthers.evaluateNotificationState(testResult: self.testResult)
```

Comment:
Shouldn't this also be checked when this view appears?

Reply:
Good point. I have refactored the warnOthers thingy into its own function in the `viewModel`
```
func updateWarnOthers() {
		self.warnOthers.evaluateNotificationState(testResult: self.testResult)
	}
```

and call the function now from the `viewWillAppear` in the controller.